portfolio/src/main/webapp/scriptMaps.js

```diff
@@ -177,4 +182,28 @@ function bounce(marker, infoWindow) {
     }
 }
 
+function addUserMarkers(map) {
+    fetch('/map-marker').then(response => response.json()).then((coords) => {
+        coords.forEach((coord) => {
+            createMarker(coord, map);
+        })
+    });
+}
+
+function addNewMarker(latLng) {
+    let latIn = latLng.lat();
+    let lngIn = latLng.lng();
+    const request = new Request(('/map-marker?lat=' + latIn + '&lng=' + lngIn), {method: 'POST'});
```

Comment:
A non-manual way of building the request is to use URLSearchParams class.  [1] for documentation.

when you fetch you can do:

fetch('/map-markers', {method: 'POST', body: params});

[1] https://developer.mozilla.org/en-US/docs/Web/API/URLSearchParams/append